tools/cythonize.py

```diff
@@ -48,6 +48,19 @@
 except NameError:
     WindowsError = None
 
+
+# Determine whether or not to enable coverage in cython files.
+linetrace = os.environ.get('linetrace', False)
+CYTHON_TRACE = str(int(bool(linetrace)))
+
+directives = {'linetrace': False}
```

Comment:
Need `'binding': True` it appears.  I got one run in `arch` for a single cython function using your setup + binding. 

https://stackoverflow.com/questions/24144931/python-line-profiler-and-cython-function

Reply:
`binding` isn't needed, sorry.